### ISSUE TITLE
lastpass shows up on the bottom of the screen

### ISSUE THREAD

**Issue by posix4e** [brave/electron#112]
* Electron version:
chromium-54
* Operating system:
Linux

```
/home/posi/brave/browser-laptop-bootstrap/src/out/Release/brave --user-data-dir=brave-development --enable-logging --v=0 /home/posi/brave/browser-laptop-bootstrap/src/browser-laptop
[26840:26840:1206/161323:ERROR:browser_main_parts.cc(173)] Could not set linux app data path
(node:26840) DeprecationWarning: Using Buffer without `new` will soon stop working. Use `new Buffer()`, or preferably `Buffer.from()`, `Buffer.allocUnsafe()` or `Buffer.alloc()` instead.
Crash reporting enabled
(node:26840) DeprecationWarning: Calling an asynchronous function without callback is deprecated.
(node:26840) DeprecationWarning: Calling an asynchronous function without callback is deprecated.
[26840:26840:1206/161326:ERROR:extension_function_dispatcher.cc(613)] Unknown Extension API - permissions.contains
[26840:26840:1206/161346:ERROR:CONSOLE(19)] "Uncaught TypeError: The first argument is the receiver and must be an object", source: extensions::SafeBuiltins (19)
[26840:26840:1206/161346:ERROR:CONSOLE(19)] "Uncaught TypeError: The first argument is the receiver and must be an object", source: extensions::SafeBuiltins (19)
[26840:26840:1206/161353:ERROR:CONSOLE(19)] "Uncaught TypeError: The first argument is the receiver and must be an object", source: extensions::SafeBuiltins (19)
[26840:26840:1206/161353:ERROR:CONSOLE(19)] "Uncaught TypeError: The first argument is the receiver and must be an object", source: extensions::SafeBuiltins (19)
WARNING: requesting non-HTTPS URL http://johnkalberer.com/favicon.ico
```

![screenshot from 2016-12-06 15-01-53](https://cloud.githubusercontent.com/assets/806363/20950166/f5af2776-bbd2-11e6-91f4-f7c8d40bd8fe.png)

**Comment by darkdh** [comment]
also happens on mac

**Comment by darkdh** [comment]
`permission.contains` error is fixed in fd6561a8a2355857f
but on the bottom problem still exists

**Comment by bsclifton** [comment]
@darkdh we are behind on the version of LastPass, just FYI

Per discussion on Slack in #extensions, version 4.1.34 is available but we are on 4.1.28. @bbondy will be updating them soon :smile:
